examples/ExampleImgPickNG/app/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "private": false,
+    "nativescript": {
+        "id": "org.nativescript.helloworldng"
+    },
+    "name": "tns-template-hello-world-ng",
+    "main": "main.js",
+    "version": "2.3.4",
+    "author": "Telerik <support@telerik.com>",
+    "description": "Nativescript Angular Hello World template",
+    "license": "BSD",
+    "keywords": [
+        "telerik",
+        "mobile",
+        "angular",
+        "nativescript",
+        "{N}",
+        "tns",
+        "appbuilder",
+        "template"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/NativeScript/template-hello-world-ng"
+    },
+    "homepage": "https://github.com/NativeScript/template-hello-world-ng",
+    "android": {
+        "v8Flags": "--expose_gc"
+    },
+    "dependencies": {
```

Comment:
@PanayotCankov Sorry, I didn't recall on this, it seems to be added to the template, because I just initialized the sample from tns create ... --ng